changelog/_unreleased/2020-09-17-support-custom-cache-folders.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Support custom cache folders
+issue: NEXT-10854
+author: Yann Karl
+author_email: y.karl@webweit.de 
+author_github: @yannick80
+---
+# Core
+*  Changed property name `cacheDir` to `projectDir`, updated write path for `dump` in `src/Core/Framework/Plugin/BundleConfigDumper.php`
+*  Changed parameter from `%kernel.cache_dir%` to `%kernel.project_dir%` in service description for `BundleConfigDumper` in `src/Core/Framework/DependencyInjection/plugin.xml`
+
```

Comment:
```suggestion
```

Reply:
line removed, sorry for missing that one